scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -150,8 +150,8 @@ export class LaneShowCmd implements Command {
       name = this.lanes.getCurrentLaneName() || DEFAULT_LANE;
     }
 
-    if (!name && remote) {
```

Comment:
@davidfirst please check this change - original error didnt really make sense, both wording and checking the remote flag

Reply:
This error should go before we set the `name` arg if not provided. So it should be like this:
```
if (!name && remote) {
      throw new Error('remote flag is not supported without lane name');
 }
 if (!name) {
      name = this.lanes.getCurrentLaneName() || DEFAULT_LANE;
    }
```

(I tested it locally, it looks good this way).

Reply:
The `--remote` flag is boolean. It's not a string of the remote-name. 
When you mention the full-id in the "name" arg, I know what remote is it. 
If the name was not provided, I don't where to find this "main".

Reply:
yep i know remote is a boolean :) 
why cant you fetch the current lane? We dont know the remote of the current lane?

Reply:
anyways, probably not in the scope here so i'm reverting that change and resolving this conversation